packages/edit-site/src/components/global-styles/header.js

```diff
@@ -27,6 +38,9 @@ function ScreenHeader( { title, description } ) {
 							icon={ isRTL() ? chevronRight : chevronLeft }
 							isSmall
 							aria-label={ __( 'Navigate to the previous view' ) }
+							onClick={ () =>
+								setEditorCanvasContainerView( undefined )
+							}
```

Comment:
This seems to now close the Style Book when navigating between blocks. Is that intentional? I'd assumed folks might want to keep the style book open while working through individual block styles. I.e. let's say you wanted to make a change to the Image block and then move on to making changes to the Gallery block. Here it winds up closing the style book:

https://github.com/WordPress/gutenberg/assets/14988353/e7c8c6de-fc25-479c-8802-8ee48752f08e

Reply:
Not intentional! Good catch. I think I was overzealous with resetting `setEditorCanvasContainerView`. Will take a look at this.